.github/merge_rules.yaml

```diff
@@ -0,0 +1,12 @@
+- name: Core Reviewers
+  patterns:
+  - '*'
+  approved_by:
+  - jansel
+  - oulgen
+  - yf225
+  mandatory_checks_name:
+  - EasyCLA
```

Comment:
You don't need/care about EasyCLA(owned by LF) here, do you? If anything, it should be Meta's CLA?